deploy/liquityStrategy.ts

```diff
@@ -58,12 +60,74 @@ const func = async function (env: HardhatRuntimeEnvironment) {
       args: [],
     });
 
-    await deploy('LiquityStabilityPool', {
+    const liquityStabilityPoolArgs = [
+      LUSDDeployment.address,
+      mockLiquityPriceFeedDeployment.address,
+    ];
+    const mockLiquityStabilityPool = await deploy('LiquityStabilityPool', {
       contract: 'MockStabilityPool',
       from: deployer,
-      args: [LUSDDeployment.address, mockLiquityPriceFeedDeployment.address],
+      args: liquityStabilityPoolArgs,
+      log: true,
+    });
+
+    const troveManagerDeploymentArgs = [
+      mockLiquityStabilityPool.address,
+      mockLiquityPriceFeedDeployment.address,
+    ];
+    const troveManagerDeployment = await deploy('TroveManager', {
+      contract: 'MockTroveManager',
+      from: deployer,
+      log: true,
+      args: troveManagerDeploymentArgs,
+    });
+
+    const mock0x = await deploy('Mock0x', {
+      contract: 'Mock0x',
+      from: deployer,
+      args: [],
       log: true,
     });
+
+    address0x = mock0x.address;
+
+    if (getNetworkName() !== 'hardhat' && getNetworkName() !== 'docker') {
+      try {
+        await env.run('verify:verify', {
```

Comment:
Why is there a try/catch around every single one of these? Do we want it to continue if one fails? Also, do they have to be in sequence instead of parallel?

Reply:
We do want to continue if one fails because the verify call to etherscan often times out. I've found the best way to deal with this is for the rest of the deployment to continue, then re-run the deployment script after so that the we re-attempt to verify contracts that timed out when verifying on the first run of the script. Hardhat recognized that the contracts were already deployed so they are not redeployed. Goes in line with hardhat deployment idempotent approach.

Reply:
I also think it's better in parallel because it makes the script take less time to execute, I also think that one verification timeout should not prevent the other verifications from taking place, because those might not time out.

Reply:
understood

it's no longer in sequence can you have another look please?

Reply:
actually one second, there are some local commits that are not reflecting here :thinking: 

![image](https://user-images.githubusercontent.com/4612986/205677289-5cf3788e-e2f7-40a6-b12e-056e4b95748a.png)

Reply:
not sure what's happening here but if you check here it has been corrected https://github.com/lindy-labs/sc_solidity-contracts/pull/214/files#diff-055ebff0adb3fef0498d8ad84ea8fce8e094abc86f87b95e8b362bf1fa75c224R96

Reply:
check my last comment regarding that